docs/docs.scss

```diff
@@ -19,6 +19,12 @@ $snippet-border: 2px solid #CCC;
 $icon-block-bg: #eee;
 $icon-block-size: 75px;
 
+.daptiv-grid{
+    .box-column {
+        border: 1px solid rgba(86, 61, 124, .15);
```

Comment:
Yes :(. I probably should have used something from colors right? BTW: this is not in style guide styles just docs styles.